pkgs/by-name/pd/pdfium-binaries/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+}:
+let
+  version = "6721";
+  src =
+    let
+      inherit (stdenv.hostPlatform) system;
+      selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
```

Comment:
This is not necessary as attempting to build a derivation on an unsupported platform already fails anyway.